packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -935,13 +935,19 @@ class RenderParagraph extends RenderBox
         final GestureRecognizer? recognizer = info.recognizer;
         if (recognizer != null) {
           if (recognizer is TapGestureRecognizer) {
-            configuration.onTap = recognizer.onTap;
```

Comment:
not related to this PR. It feels weird to me configuration.onTap is a nullable value but will crash if you really give it null.

Reply:
Yeah, the problem is that the `onTap` getter must be nullable, but the setter is not really and dart doesn't allow specifying that...

Reply:
Do we have use case where we want to reset configuration.onTap back to null?